packages/block-editor/src/components/navigable-toolbar/index.js

```diff
@@ -147,6 +154,7 @@ function useToolbarFocus(
 function NavigableToolbar( {
 	children,
 	focusOnMount,
+	useKeyboardFocusShortcut = true,
```

Comment:
Could we rename this to `shouldUseKeyboardFocusShortcut` as seen below? `use*` variable often suggests a hook in React and that might cause some confusion. `shouuld*` also has the benefit of suggesting the variable to be a boolean too.